### PR TITLE
Course info displayer 

### PR DESCRIPTION
**Pull request by lorena-b**
# Description

Extracting course description from the API and storing it in a course object, then displaying the course information to the user. 

Demo: 
image

## Type of change

- [X] New feature 

# How Has This Been Tested?

Manual testing for display, test suite for `CourseCreator` to ensure the correct description is being stored. 

# Does this PR follow the rubric?

NGL not sure which layer the `CourseInfoDisplay` class would be in and also if it follows SRP but I wanted to avoid bloating the CLI class 

# Checklist:

- [x] I ran `./gradlew spotlessApply`
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I ran `./gradlew build` and the gradle build succeeded
- [ ] I have not violated any design principles

### CODE REVIEW

File: src/main/java/util/CourseInfoDisplay.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        }
+        System.out.println(
+                "---------------------------------------------------------------------------------------------");
+        System.out.println(
+                "Exclusions: "
+                        + ConsoleColours.BLUE
+                        + course.getExclusions().toString()
+                        + ConsoleColours.RESET);
+        System.out.println(
+                "Corequisites: "
```
Comment (roryhofland) [inline src/main/java/util/CourseInfoDisplay.java:98]: When a course has a recommended corequisite, the program prints `Corequisites: [Recomm]`. An example would be MAT332F.

**Review by roryhofland — CHANGES_REQUESTED** [review]
Looks good other than the one small comment. I think it would also be good if the user could go from reading the courses to making a schedule with them, but its probabily not worth the effort adding that in.

File: src/main/java/entities/Course.java [hunk 2]
```diff
@@ -20,21 +21,24 @@
...
      */
     public Course(
             String courseId,
             List<Section> lectures,
             List<Section> tutorials,
             char session,
             List<String> exclusions,
-            List<String> corequisites) {
+            List<String> corequisites,
+            String courseDesc) {
```
Comment (EclipseIsDead) [inline src/main/java/entities/Course.java:33]: this needs to update in our tests as well

**Review by EclipseIsDead — COMMENTED** [review]
nice stuff

**Review by EclipseIsDead — APPROVED** [review]
lgtm

File: src/main/java/workers/CourseCreator.java [hunk 3]
```diff
@@ -138,12 +144,12 @@ private static Double getProfessorRating(JsonObject lecture) {
      * @param value a String that corresponds to all the exclusions for a course
      * @return an ArrayList of course names
      */
-    public static List<String> getCourseExclusions(String value, String courseID) {
+    public static List<String> getCourseExclusions(String value) {
         List<String> shortenedCodes = new ArrayList<>();
         try {
             extractCodes(value, shortenedCodes);
         } catch (Exception IndexOutOfBoundsException) {
-            System.out.println("The course " + courseID + " has no exclusions (empty string)");
```
Comment (kenneth-miura) [inline src/main/java/workers/CourseCreator.java:146]: WOOOOOOOOOOOOOOOO

**Review by kenneth-miura — APPROVED** [review]
lit. lgtm

**Pull request merged**